lib/ansible/modules/crypto/openssl_csr.py

```diff
@@ -272,6 +383,8 @@ def dump(self):
             'csr': self.path,
```

Comment:
nit: can you:

 * rename `csr` to `filename` to matches privatekey and publickey module. So as the user, one would always know what is the return parameter s/he needs to look at.
 * add 'privatekey': self.privatekey_path, so a user would know which privatekey has been used to generate that csr